snmp/datadog_checks/snmp/data/profiles/_hp-compaq-health.yaml

```diff
@@ -0,0 +1,190 @@
+metrics:
```

Comment:
Should we embed a link to the "official" source of this MIB? (I think we mentioned that eg Circitor was kind of out of date.)

Maybe also with a short description of this file?

```suggestion
# Generic device health metrics for HP devices.
# The latest version of these MIBs can be downloaded here:
# https://support.hpe.com/hpsc/swd/public/detail?swItemId=MTX_53293d026fb147958b223069b6

metrics:
```

Reply:
Sure, updated!